src/main/webapp/app/shared/pipes/posting-content-markdown-linebreak.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+    name: 'postingContentMarkdownLinebreak',
+})
+export class PostingContentMarkdownLinebreakPipe implements PipeTransform {
+    /**
+     * Replaces a markdown linebreak into an html tag to ensure that further transformation in htmlForMarkdownPipe will not remove linebreaks
+     * @param {string} markdown the original markdown text
+     */
+    transform(markdown?: string): string {
+        if (markdown) {
+            return markdown.replace(/\n/gm, '<br>');
+        } else {
+            return '';
+        }
+    }
+}
```

Comment:
[nit] Does it only make sense in the context of postings or for any markdown editor?
Should we extend it to other usages of the markdown editor?

Reply:
The issue only occurs for postings, I had to systematically manipulate the `<p>` tags in posting contents to allow rendering a sentence such as `Check post #58 to find further information` in one line instead of having a line break before and after the post reference. Technically, it's an anchor tag between to paragraphs in a row `<p> Check post </p><a>#58</a><p> to find further information</p>`. Long story short, it's not required in other use cases :)